setup.py

```diff
@@ -12,9 +12,9 @@ def read_file(fname):
     long_description=read_file('README'),
     keywords='bioinformatics',
     install_requires=[
-        'tornado==4.2.1', 
-        'PyYAML==3.11', 
-        'requests==2.7.0'
+        'tornado==4.2.1',
+        'PyYAML==3.13',
+        'requests==2.20.0'
```

Comment:
what is the interplay between `setup.py` and `requirements` files? I've sometimes seen (and copied) implementations where `setup.py` parses the requirements file into the `install_requires` list.